### ISSUE TITLE
File could not be found. 

### ISSUE THREAD

**Issue by SaadBinShahid** [estebistec/django-twitter-bootstrap#21]
I'm using pipeline. 
I tried, explicitly over-riding the font file path. But no luck.
ValueError: The file 'twitter_bootstrap/fonts/glyphicons-halflings-regular.woff2' could not be found with <pipeline.storage.PipelineCachedStorage object at 0xa8e48ec>.

**Comment by estebistec** [comment]
does that file appear in the installed location of dj-tw-bs?

**Comment by SaadBinShahid** [comment]
No, I created an empty file with the same name and extension. Error seems
to go away.

Regards,
Saad Bin Shahid | Software Engineer | www.saadbinshahid.com |

**Comment by estebistec** [comment]
Yes, but surely you don't get the desired icons? That doesn't sound right at all. I need to go check the packaging...

**Comment by SaadBinShahid** [comment]
No, I did not get my desired icons. please look into it. Thanks.!

Regards,
Saad Bin Shahid | Software Engineer | www.saadbinshahid.com |

**Comment by estebistec** [comment]
Oh, you have `woff2` as the file extension. It's just `woff`.

**Comment by SaadBinShahid** [comment]
I know that :) I checked. There is an .woff2 empty file in the repo
@ bootstrap/fonts/glyphicons-halflings-regular.woff2

That's why I added a empty file. The file is also being used in the CSS and
some other places. Hence raising the issue when the PIPELINE try to
pre-process it.

Regards,
Saad Bin Shahid | Software Engineer | www.saadbinshahid.com |

**Comment by estebistec** [comment]
Okay. I see. Looking at the currently supported version of bootstrap: https://github.com/twbs/bootstrap/tree/16dbdbd7a2c6cfa3be4e5dcc52249e577c02c84a/fonts, there is no `woff2`. It's been added in a recent version. So really this isn't a bug, I just need to release new wrapper packages for latest bootstrap (which will include `woff2`).

However, I've been meaning to give this repo and package the correct name (see #13). So rather than just release the next versions (one of which would include the `woff2`), I'd like to close this issue and complete #13 and release subsequent versions.

**Comment by SaadBinShahid** [comment]
Great.! Thanks.

Regards,
Saad Bin Shahid | Software Engineer | www.saadbinshahid.com |
